gamecenter/public/views.py

```diff
@@ -5,12 +5,13 @@
 blueprint = Blueprint('public', __name__, static_folder="../static")
 
 
-@blueprint.route("/", methods=["GET"])
+@blueprint.route("/")
 def home():
-    # Handle logging in
+    """Return the home page"""
```

Comment:
I'm not a fan of these kinds of comments, but if I remember correctly, we've had this convo before so nbd leaving them.